src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2130,12 +2130,13 @@ namespace RCT1
             }
 
             // Awards
-            for (int32_t i = 0; i < Limits::MaxAwards; i++)
+            auto& awards = GetAwards();
+            for (auto& src : _s4.awards)
             {
-                rct12_award* src = &_s4.awards[i];
-                Award* dst = &gCurrentAwards[i];
-                dst->Time = src->time;
-                dst->Type = src->type;
+                if (src.time != 0)
+                {
+                    awards.push_back(Award{ src.type, src.time });
```

Comment:
```suggestion
                    awards.emplace_back(src.type, src.time);
```

Reply:
urgh. C++20 feature unfortunately. https://stackoverflow.com/questions/43394841/emplace-an-aggregate-in-stdvector

Reply:
Argh, that's a bummer.